pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/forward/ForwardIndexType.java

```diff
@@ -56,6 +56,7 @@
 public class ForwardIndexType
     extends AbstractIndexType<ForwardIndexConfig, ForwardIndexReader, ForwardIndexCreator>
     implements ConfigurableFromIndexLoadingConfig<ForwardIndexConfig> {
+  public static final String INDEX_DISPLAY_NAME = "forward";
```

Comment:
Can we build these by getting prefix of SandardIndexes IDs? i.e. forward_index ==> forward, etc. that eliminates the need of having two distributed set of ids and makes this more maintainable

Reply:
Not all ids follow the same name convention. Also, not all indexes may be in StandardIndexes. The ability to add new indexes at runtime was the reason to implement the index SPI in the first term.